src/integrationTest/resources/db/migration/V1_9999__integrationTestData.sql

```diff
@@ -1,228 +0,0 @@
-INSERT INTO courthouse (cth_id, courthouse_name, courthouse_code, created_ts, last_modified_ts) VALUES (1, 'SWANSEA', 457, '2023-05-06 00:00:00+00', '2023-06-14 16:45:09.998033+00');
```

Comment:
i would say that this file is quite useful for local testing, even if its not being used for integration tests. Would we be able to move it somewhere instead of deleting?

Reply:
done